qbicsoftware/data-manager-app
# Feature/dm 407 provide UI for sample measurement information

**What was changed**
This PR provides a UI implementation for showing the measurement data with lazy loading.

**ToDo**
1.) Decide on the properties to be filtered 
2.) Introduce frontend records hosting these properties to avoid leaking "entity" information
3.) Update Measurement validation in uploadDialog

**How to test**
A) Go to a measurement page without registered measurements --> disclaimer is shown
B) Go to the measurement main component for project `3997943c-4a08-4990-8df9-312230118445` and its experiment `68e55d0f-053e-4ac1-a346-2e4edc73e0aa` to see the registered proteomics measurements
